src/platform/graphics/webgl/webgl-graphics-device.js

```diff
@@ -787,10 +786,24 @@ class WebglGraphicsDevice extends GraphicsDevice {
         this.backBuffer.impl.suppliedColorFramebuffer = frameBuffer;
     }
 
+    // Update framebuffer format based on the current framebuffer, as this is use to create matching multi-sampled framebuffer
+    updateBackbufferFormat(framebuffer) {
+        const gl = this.gl;
+        gl.bindFramebuffer(gl.FRAMEBUFFER, framebuffer);
+        const alphaBits = this.gl.getParameter(this.gl.ALPHA_BITS);
```

Comment:
any performance worries here? i thought getParameter was real slow.

Reply:
this happens when entering / exiting XR .. so no worries? We do this already when the engine starts up. It's not that slow.

Reply:
oic, thought this was per frame 👍🏻